### PR TITLE
[fix]MON-13067-install-remove-default-value-for-colors-in-options-table-for-develop

### PR DESCRIPTION
**Pull request by emabassi-ext**
## Description

Actually on a fresh install, we insert values for colors in database. (in options table). Those values seems not used anymore and should be removed.
**Fixes** # MON-13067

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x (master)

<h2> How this pull request can be tested ? </h2>
Install a new Centreon central server

Execute following request on ‘centreon’ (configuration) database:

SELECT * FROM options WHERE options.key like "color_%"
No result must be found
#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by jeremyjaouen — DISMISSED** [review]

**Review by jeremyjaouen — DISMISSED** [review]

**Review by dmyios — DISMISSED** [review]

File: www/install/sql/centreon/Update-DB-2.8.6.sql [hunk 1]
```diff
@@ -2,13 +2,3 @@
...
-
--- Change state colors
-UPDATE `options` SET `value` = '#88b917' WHERE `key` in ('color_up', 'color_ok');
-UPDATE `options` SET `value` = '#e00b3d' WHERE `key` in ('color_down', 'color_critical', 'color_host_down');
-UPDATE `options` SET `value` = '#818285' WHERE `key` in ('color_unreachable', 'color_host_unreachable');
-UPDATE `options` SET `value` = '#ff9a13' WHERE `key` = 'color_warning';
-UPDATE `options` SET `value` = '#2ad1d4' WHERE `key` = 'color_pending';
-UPDATE `options` SET `value` = '#ae9500' WHERE `key` = 'color_ack';
-UPDATE `options` SET `value` = '#bcbdc0' WHERE `key` = 'color_unknown';
-UPDATE `options` SET `value` = '#cc99ff' WHERE `key` = 'color_downtime';
```
Comment (kduret) [inline www/install/sql/centreon/Update-DB-2.8.6.sql]: this upgrade script must not be touched

**Review by kduret — CHANGES_REQUESTED** [review]
values must be deleted from database in upgrade script to 22.10.0

File: www/install/sql/centreon/Update-DB-2.8.6.sql (hunk 1 above)
Comment (emabassi-ext) [reply www/install/sql/centreon/Update-DB-2.8.6.sql]: done!

**Review by kduret — COMMENTED** [review]
values must be deleted from database in upgrade script to 22.10.0-beta.1

File: www/install/php/Update-22.10.0-beta.1.php [hunk 2]
```diff
@@ -31,7 +31,7 @@
     $pearDB->beginTransaction();
 
     $errorMessage = "Unable to delete 'oreon_web_path' option from database";
-    $pearDB->query("DELETE FROM `options` WHERE `key` = 'oreon_web_path'");
+    $pearDB->query("DELETE FROM `options` WHERE `key` = 'oreon_web_path' OR `key` like 'color_%'");
```
Comment (kduret) [inline www/install/php/Update-22.10.0-beta.1.php]:
```suggestion
    $pearDB->query("DELETE FROM `options` WHERE `key` = 'oreon_web_path' OR `key` LIKE 'color_%'");
```

File: www/install/php/Update-22.10.0-beta.1.php [hunk 3]
```diff
@@ -31,7 +31,7 @@
     $pearDB->beginTransaction();
 
     $errorMessage = "Unable to delete 'oreon_web_path' option from database";
```
Comment (kduret) [inline www/install/php/Update-22.10.0-beta.1.php]:
```suggestion
    $errorMessage = "Unable to delete 'oreon_web_path' and color options from database";
```

**Review by kduret — APPROVED** [review]

**Review by wtermellil — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Pull request merged**